docs/developer/release/7-test-release.md

```diff
@@ -1,9 +0,0 @@
-# Test Release
-
-Validate the new version is working by running it incrementally in all dogfooding clusters.
-
-## Steps
-
-1. Validate performance metrics are consistent with the prior version.
-
-2. Validate components are healthy.
```

Comment:
why delete this step? It's important not to forget to update our infra.

Reply:
It's very ambiguous how you'd do it. For example:
* Which components should this be done for?
* Which config should be used?
* Which platforms - Windows, Mac, Linux, BSD? 

Ultimately, is this not a step which can be automated by the CI? It sounds like something which should be done by the unit tests.

Reply:
we can't test everything but it's an additional layer that has the benefit of testing real use cases at scale. 
For example Alloy 1.3 rc showed an increased rate of OOO errors and that's something we caught because we tested the RC in our infra

Reply:
Ok, this is the first time I realise that this is a reminder step to update Grafana's internal environments :) I thought this is about testing locally. I'll update the step instructions to mention it more clearly. Thank you for the feedback!

Reply:
Actually, it already says "dogfooding clusters" - idk how I missed it 😅  I guess I didn't read the description and just went straight to the steps.